app/uk/gov/hmrc/helptosavefrontend/controllers/NewApplicantUpdateEmailAddressController.scala

```diff
@@ -63,6 +63,7 @@ class NewApplicantUpdateEmailAddressController @Inject() (val sessionCacheConnec
           userInfo ⇒
             sendEmailVerificationRequest(
               email,
+              Name(userInfo.forename, userInfo.surname),
```

Comment:
don't we need `middleName` at all ?

Reply:
in the JSON we send to NS&I the middle name is not one of the fields, so I guess we don't care about the middle name